pkg/tinydtls/contrib/sock_dtls.c

```diff
@@ -376,6 +436,26 @@ static ssize_t _copy_buffer(sock_dtls_t *sock, void *data, size_t max_len)
     if (buflen > max_len) {
         return -ENOBUFS;
     }
+#if SOCK_HAS_ASYNC
+    if (sock->buf_ctx != NULL) {
+        memcpy(data, buf, sock->buflen);
+        _check_more_chunks(sock->udp_sock, (void **)&buf, &sock->buf_ctx,
+                           &remote->ep);
+        if (sock->async_cb && cib_avail(&sock->mbox.cib)) {
```

Comment:
Why the check for `sock->mbox.cib`?

Reply:
To check if there is actually another message in the mbox?

Reply:
I guess the mbox API could use a `static inline` wrapper function then, so API users don't have to deal with implementation details.

A `/* check if another message is in the mbox */` would be good regardless to ease navigating the code.

Reply:
> I guess the mbox API could use a `static inline` wrapper function then, so API users don't have to deal with implementation details.

This is the check we use everywhere for checking if something is in an `mbox`... Be my guest to provide an amendment to `core/include/mbox.h` :-)